packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -1685,7 +1685,7 @@ - (NSDictionary *)firebaseUserToDict:(FIRUser *)user {
 - (NSArray<NSMutableDictionary *> *)convertMultiFactorData:(NSArray<FIRMultiFactorInfo *> *)hints {
   NSMutableArray *enrolledFactors = [NSMutableArray array];
 
-  for (FIRPhoneMultiFactorInfo *hint in hints) {
+  for (FIRMultiFactorInfo *hint in hints) {
```

Comment:
TBH I'm not 100% sure on this

Reply:
Caused a build failure. I think it would be more accurate to loop over `FIRMultiFactorInfo` and then branch based on the specific type, but this will work since TOTP doesn't have any additional properties :shrug: